src/components/Textfields/TextFieldBase/index.tsx

```diff
@@ -16,6 +18,7 @@ export interface TexfieldBaseProps extends MaterialTextFieldProps {
     errorMessage?: string;
     inputRef?: React.RefObject<HTMLInputElement>;
     InputProps?: Partial<OutlinedInputProps>;
+    onChange?: ChangeEventHandler<HTMLInputElement | HTMLTextAreaElement>;
 }
```

Comment:
Thanks for add this props for my component

Reply:
I don't know why TextFieldProps doesn't have that prop @@